Terminal-iOS/MyStudy/StudyDetail/ViewComponent/MyApplyStudyInfo/MyApplyStudyInfoInteractor.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  MyApplyStudyInfoInteractor.swift
+//  Terminal-iOS
+//
+//  Created by 정재인 on 2021/01/29.
+//  Copyright © 2021 정재인. All rights reserved.
+//
+
+import Foundation
+
+class MyApplyStudyInfoInteractor: MyApplyStudyInfoInteractorInputProtocol {
+    var presenter: MyApplyStudyInfoInteractorOutputProtocol?
+    var remoteDataManager: MyApplyStudyInfoRemoteDataManagerInputProtocol?
+    var applyID: Int?
+    var studyID: Int?
+    
+    func deleteApply() {
+        if let applyIdentifier = applyID, let studyIdentifier = studyID {
+            remoteDataManager?.deleteApply(studyID: studyIdentifier, applyID: applyIdentifier)
+        }
+    }
+}
+
+extension MyApplyStudyInfoInteractor: MyApplyStudyInfoRemoteDataManagerOutputProtocol {
+    func retriveDeleteApplyResult(result: Bool, message: String) {
+        switch result {
+        case true:
+            presenter?.retriveDeleteApplyResult(result: result, message: message)
+        case false:
+            presenter?.retriveDeleteApplyResult(result: result, message: message)
+        }
```

Comment:
```suggestion
 presenter?.retrieveDeleteApplyResult(result: result, message: message)
```
이건 질문인데 이렇게 한 이유가 있을까요?? 다른 이유가 있어서 일부러 이렇게 작성한건지

Reply:
음 이게뭔지 모르겠네요 스위치문 밑에 
```presenter?.retrieveDeleteApplyResult(result: result, message: message)```
이렇게 써져있는 건가요?
의도한 건아니고 제 로컬에도 되어있지 않습니다